WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/AdaptiveModalContainer.swift

```diff
@@ -89,6 +93,9 @@ struct AdaptiveModalContainer<PrimaryView: View, SecondaryView: View>: View {
                 .navigationViewStyle(.stack)
                 .frame(minWidth: 400)
             }
+            .task {
+                isShowingSecondaryView = true
+            }
```

Comment:
Just curious: since `.task` is meant for async actions before the view appears, in this case wouldn't be the same as using `.onAppear` ?

Reply:
Yes, it would be the same. I'll change it because it's slightly more descriptive.